data/eos-update-server.conf

```diff
@@ -0,0 +1,4 @@
+# This is an example configuration file showing default values.
+# To use it, copy it to /etc/eos-updater/eos-update-server.conf and edit it.
```

Comment:
This is not entirely true; the code will actually read the configuration file. Perhaps this example file should be installed with the default values commented out? IIRC that's what systemd does.